src/stcal/jump/jump.py

```diff
@@ -471,8 +473,7 @@ def detect_jumps(
     data /= gain_2d
     err /= gain_2d
     readnoise_2d /= gain_2d
-
-    # Return the updated data quality arrays
+     # Return the updated data quality arrays
```

Comment:
There is an extra space before the comment.

Reply:
fixed